package/utils.py

```diff
@@ -72,3 +72,13 @@ def normalize_license(license: str):
             return "Custom"
         return stripped_license
     return "UNKNOWN"
+
+
+def extract_documentation_url_from_markdown(description):
```

Comment:
Calling this function with `description="[Documentation](This is not a URL)"` will return an invalid URL. If we **really** need to extract the documentation URL from the `description` perhaps we could use some kind of `regex`? 

**Note:** A really large description filed might cause some performance issues.

Reply:
Django has its own `URLValidator` Can we use that? URL regex gets pretty complex.

Reply:
I did a simple test. How can I improve it?
![image](https://github.com/djangopackages/djangopackages/assets/62690310/1ad295ed-9697-4e5a-8954-f9cdca322b42)